.gitignore

```diff
@@ -14,6 +14,7 @@ exports/*
 .coverage
 htmlcov
 gamestonk_terminal/reports/stored/*
+gamestonk_terminal/portfolio/portfolios/*
```

Comment:
<img width="903" alt="Screenshot 2021-10-31 at 09 42 31" src="https://user-images.githubusercontent.com/25267873/139576574-f331dcf0-0ec2-4acb-bfe2-276fe04f90d2.png">

Can we have at least 1 example here that users can use to understand how to build their portfolios?